todoApp/samples/javascript/todoItemSamples.js

```diff
@@ -0,0 +1,78 @@
+const { TodoClient } = require("@notabrand/todo-non-branded");
+
+// Load the .env file if it exists
+require("dotenv").config();
+
+const endpoint = process.env["ENDPOINT"] || "<endpoint>";
+
+const client = new TodoClient(
+  endpoint,
+  { key: "dummy" },
+  { allowInsecureConnection: true }
+);
+
+async function main() {
+  const item = {
+    title: "Buy milk",
+    createdBy: 1,
+    assignedTo: 10,
+    description: "Need to buy milk",
+    status: "InProgress",
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    id: 1
+  };
```

Comment:
should we move this down to the place that we are about to use it?
It is weird that we define it here, but the next line is not using this but creates a user.

Reply:
updated